Fixes the following issue with glibc 2.36:

Error occurred during initialization of boot layer
java.lang.AssertionError: java.net.URISyntaxException: Illdk-k.internal.jvmstat/
Caused by: java.net.URISyntaxException: Illegal character in path at index 32: file:///builddir/jdk11u-jdk-11-2er-release/jdk/modules/jdk.internal.jvmstat/
make[3]: *** [ExplodedImageOptimize.gmk:41: /builddir/jdk11u-jdk-11-27/build/linux-x86_64-normal-server-release/jdk/_packages_attribute.done] Error 1
make[2]: *** [make/Main.gmk:347: exploded-image-optimize] Error 2

From 789b6089954d04a0222434916c93b1ed33c629c8 Mon Sep 17 00:00:00 2001
From: Vladimir Kozlov <kvn@openjdk.org>
Date: Mon, 20 Aug 2018 15:10:20 -0700
Subject: [PATCH] 8209735: Disable avx512 by default

Reviewed-by: iveresov, ecaspole
---
 src/hotspot/cpu/x86/globals_x86.hpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/hotspot/cpu/x86/globals_x86.hpp b/src/hotspot/cpu/x86/globals_x86.hpp
index 423e1b1e478..49073f1834a 100644
--- a/src/hotspot/cpu/x86/globals_x86.hpp
+++ b/src/hotspot/cpu/x86/globals_x86.hpp
@@ -119,7 +119,7 @@ define_pd_global(bool, ThreadLocalHandshakes, false);
   product(bool, UseStoreImmI16, true,                                       \
           "Use store immediate 16-bits value instruction on x86")           \
                                                                             \
-  product(intx, UseAVX, 3,                                                  \
+  product(intx, UseAVX, 2,                                                  \
           "Highest supported AVX instructions set on x86/x64")              \
           range(0, 99)                                                      \
                                                                             \
